### ISSUE TITLE
Uncaught TypeError: Cannot read property 'onDidUpdate' of undefined

### ISSUE THREAD

**Issue by joshuapowell** [atom-minimap/minimap-find-and-replace#8]
[Enter steps to reproduce below:]

1. Hover over the mini-map and make sure that the `find-and-replace` option is enabled
2. Press CMD+F to activate the Find Utility
3. Receive warning on the screen

<img width="452" alt="screen shot 2015-08-17 at 9 57 08 am" src="https://cloud.githubusercontent.com/assets/569351/9306445/5839c3b8-44c6-11e5-9016-12cf9032d095.png">
<img width="454" alt="screen shot 2015-08-17 at 9 57 02 am" src="https://cloud.githubusercontent.com/assets/569351/9306446/5a03d792-44c6-11e5-929a-36c85dba8636.png">

**Atom Version**: 1.0.7
**System**: Mac OS X 10.10.5
**Thrown From**: [minimap-find-and-replace](https://github.com/atom-minimap/minimap-find-and-replace) package, v4.2.1

### Stack Trace

Uncaught TypeError: Cannot read property 'onDidUpdate' of undefined

```
At /Users/***/.atom/packages/minimap-find-and-replace/lib/minimap-find-results-view.coffee:9

TypeError: Cannot read property 'onDidUpdate' of undefined
  at new MinimapFindResultsView (/Users/***/.atom/packages/minimap-find-and-replace/lib/minimap-find-results-view.coffee:9:32)
  at MinimapFindAndReplaceBinding.module.exports.MinimapFindAndReplaceBinding.activate (/Users/***/.atom/packages/minimap-find-and-replace/lib/minimap-find-and-replace-binding.coffee:49:20)
  at MinimapFindAndReplaceBinding.activate (/Users/***/.atom/packages/minimap-find-and-replace/lib/minimap-find-and-replace-binding.coffee:1:1)
  at MinimapFindAndReplaceBinding.module.exports.MinimapFindAndReplaceBinding.activatePlugin (/Users/***/.atom/packages/minimap-find-and-replace/lib/minimap-find-and-replace-binding.coffee:33:6)
  at Main.module.exports.PluginManagement.updatesPluginActivationState (/Users/***/.atom/packages/minimap/lib/mixins/plugin-management.coffee:89:14)
  at Main.module.exports.PluginManagement.togglePluginActivation (/Users/***/.atom/packages/minimap/lib/mixins/plugin-management.coffee:70:6)
  at /Users/***/.atom/packages/minimap/lib/minimap-quick-settings-element.coffee:134:22
  at HTMLLIElement.<anonymous> (/Users/***/.atom/packages/minimap/lib/minimap-quick-settings-element.coffee:139:7)

```

### Commands

```
     -0:18.5.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:18.5.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:17.8.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -0:17.2.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:17.2.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:14.7.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -0:14.3.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
  2x -0:14.3.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:10.2.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:10.2.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:05.2.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -0:04.7.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:04.7.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:04.1.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -0:03.4.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:03.4.0 core:cancel (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "base16-tomorrow-dark-theme"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autoclose-html, v0.18.0
autocomplete-python, v0.5.3
jshint, v1.3.10
minimap, v4.12.3
minimap-find-and-replace, v4.2.1
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.0
minimap-selection, v4.3.0

# Dev
No dev packages
```

**Comment by abe33** [comment]
Hi @joshuapowell, and thanks for the report, but it seems I can't reproduce it, is it systematic or is there some other conditions?

**Comment by clementlamoureux** [comment]
I've got the same, it's systematic for me

**Comment by randallb** [comment]
It seems to only happen after I've left it open for some amount of time.

**Comment by 1j01** [comment]
It's happening for me without hovering over the minimap, but I can't reproduce it in a new window.

**Comment by abe33** [comment]
Hi guys, I've just published version v4.3.0 that is a partial rewrite of the plugin (hence the minor bump), let me know if the new version gives better results in your cases. 

The main change is that it no longer relies on the `find-and-replace` model but directly on the markers that are created in the text-editor. It simplifies a lot of things and should be more reliable.

**Comment by clementlamoureux** [comment]
It seems that this issue disappeared with the new version, thanks !
